icons/bottle-spray.svg

```diff
@@ -0,0 +1,20 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 2h.01" />
+  <path d="M20 5.5h.01" />
+  <path d="M22 9h.01" />
+  <path d="M10 8a2 2 0 0 1 2-2h3c.6 0 1-.4 1-1V3c0-.6-.4-1-1-1H5C3.3 2 2 3.3 2 5c0 .6.4 1 1 1h1a2 2 0 0 1 2 2v2l-2.3 2.3c-.4.4-.7 1.1-.7 1.7v6a2 2 0 0 0 2 2h6a2 2 0 0 0 2-2v-6c0-.6-.3-1.3-.7-1.7L10 10Z" />
+  <path d="M12 2v4" />
+  <path d="M14 6c0 2 0 3 2 4" />
+  <path d="M8 9v9" />
```

Comment:
Removed straw due to violating 2px rule.
![icon](https://github.com/lucide-icons/lucide/assets/25524993/6cb6bdb9-ddc4-4706-a3bc-e7adf85093f6)

```suggestion
```

Reply:
Yeah, it also doesn't scale well: 
![image](https://github.com/lucide-icons/lucide/assets/17746067/31431bb8-0f57-4ec2-a0df-223f56821c73)